README.md

```diff
@@ -209,6 +218,10 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [QMASM](https://github.com/lanl/qmasm) - A quantum macro assembler for D-Wave's quantum annealers.
 - [Quil](https://arxiv.org/abs/1608.03355) - An open hybrid quantum/classical instruction set currently used by Rigetti. [Parser](https://github.com/rigetticomputing/pyquil/tree/master/pyquil/_parser)
 
+## Quantum Control
```

Comment:
Could you please sort this category by programming language such that it follows the same pattern as the other sections?